tests/e2e/steps/steps_def.go

```diff
@@ -753,6 +782,84 @@ func (step *Step) validateAuthorizationProxyServerNotInstalled(cr csmv1.Containe
 	return nil
 }
 
+func (step *Step) authProxyServerPrereqs(cr csmv1.ContainerStorageModule) error {
+	fmt.Println("=== Creating Authorization Proxy Server Prerequisites ===")
+
+	cmd := exec.Command("kubectl", "get", "ns", "authorization")
+	err := cmd.Run()
+	if err == nil {
+		cmd = exec.Command("kubectl", "delete", "ns", "authorization")
+		b, err := cmd.CombinedOutput()
+		if err != nil {
+			return fmt.Errorf("failed to delete authorization namespace: %v\nErrMessage:\n%s", err, string(b))
+		}
+	}
+
+	cmd = exec.Command("kubectl", "create",
+		"ns", "authorization",
+	)
+	b, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to create authorization namespace: %v\nErrMessage:\n%s", err, string(b))
+	}
+
+	cmd = exec.Command("kubectl", "apply",
+		"--validate=false", "-f",
+		"https://github.com/jetstack/cert-manager/releases/download/v1.11.0/cert-manager.crds.yaml",
```

Comment:
The version, v1.11.0, could be a constant at the top of the file to easily change later.

Reply:
made it a constant